src/store/mode/__tests__/reducer-test.ts

```diff
@@ -0,0 +1,14 @@
+import modeReducer from '../reducer';
+import { toggleAnnotationModeAction } from '../actions';
+
+describe('store/mode/reducer', () => {
+    describe('toggleAnnotationModeAction', () => {
+        test.each([
```

Comment:
I usually prefer to use the template string column notation with multiple values, since it makes it easier to read. Up to you, though.